### PR TITLE
add exercise 14 files

### PR DESCRIPTION
**Pull request by ElizaAlimasa**
please review my files, thank you!

### CODE REVIEW

File: alimasa_eliza/countries.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <link href="countries_style.css" rel="stylesheet" />
+</head>
+<body>
+    <div class="countries">
+        <h1>Countries of the World</h1>
+        <div>
+            <input type="text" id="country_input" 
+            placeholder="Please enter a country">
+            <button onclick="countrySearch()">Search</button>
+        </div>
```
Comment (Tapispisan-YvesJohn) [inline alimasa_eliza/countries.html]: Add a few indent spaces if the element has multiple lines. 
```suggestion
        <div>
            <input type="text" id="country_input" 
                   placeholder="Please enter a country">
            <button onclick="countrySearch()">Search</button>
        </div>
```

**Review by Tapispisan-YvesJohn — CHANGES_REQUESTED** [review]
Your country button is in an awkward position, move it so that it has its own div or add it to your works. 
image

Other than that and a minor indentation mistake your code looks good.

**Comment by ElizaAlimasa** [comment]
Hello, I've made some changes on my files. Please review my code again, Thank you!

**Review by Tapispisan-YvesJohn — APPROVED** [review]
Looks good to me, you can merge it now

**Comment by ElizaAlimasa** [comment]
Thank you!

**Pull request merged**